features/step_definitions/whatsapp_steps.rb

```diff
@@ -1,21 +1,35 @@
+When(/^I visit the category ([\w\-]+) in (English|Welsh)$/) do |category, language|
+  locale = case language
```

Comment:
Pretty subjective, but could use one line here: `locale = language == 'Welsh' ? 'cy' : 'en'`

Reply:
nice! will change it!